### ISSUE TITLE
Warning & Suggestion code instead of solely verbal feedback

### ISSUE THREAD

**Issue by dominguesgm** [dropbox/zxcvbn#293]
In order to work well with localized web pages/apps, one must currently map the written feedback to their translations.

For this library to provide a better experience for developing in different locales, I think it would be a good idea to add `feedback.suggestion_code` and `feedback.warning_code` which could be simpler, more concise and less prone to change which would break translations.

For example, `Add another word or two. Uncommon words are better` could have code `add_words` or `Avoid repeated words and characters` could have code `avoid_repeated`.

This would be a less prone to failure way to enable mapping custom feedback.

**Comment by MuhammedZakir** [comment]
This project is unmaintained by Dropbox for a long time now.

There is an active fork with i18n support. See https://github.com/dropbox/zxcvbn/issues/290#issuecomment-687847476.

**Comment by dominguesgm** [comment]
Yep, only noticed the last commit date after submitting the issue 😞 Thanks for the suggestion though
